### ISSUE TITLE
Accessibility scanning with html_codesniffer as its runner fails with this error while it is working with axe runner.

### ISSUE THREAD

**Issue by danghailuong1100** [pa11y/pa11y#708]
**This repository's issues are reserved for feature requests, bug reports, and other issues with the Pa11y library itself.**

If you need help integrating Pa11y with your application, we recommend using [Stack Overflow](https://stackoverflow.com/questions/tagged/pa11y).

For a bug report, please use the template below. To keep the backlog clean and actionable, issues may be immediately closed if they do not follow one this template.

---

## Environment

```
(please run `pa11y --environment` and paste the output here)
```
